commit a10fa942c4e1dd00d22a05dcf4c70806325df648 (refs/changes/38/238/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-08-05T11:39:50-07:00 (3 years, 2 months ago)
    
    DOCKER-900: update sdc-docker-build module to fix slow docker builds

diff --git a/package.json b/package.json
index 6de4e41..91861e9 100644
--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "restify": "4.0.0",
     "rimraf": "2.4.3",
     "sdc-clients": "~9.1",
-    "sdc-docker-build": "git+ssh://git@github.com:joyent/sdc-docker-build.git#f26f6de19a3860ddb8f7f57848903cf25cc34b8d",
+    "sdc-docker-build": "git+https://github.com/joyent/sdc-docker-build.git#2c5cafc2275587ab6b3f714cce3f6f38cfe9a1a9",
     "semver": "2.2.1",
     "shell-escape": "^0.2.0",
     "sprintf": "0.1.1",
